addons/csi-vmware-cloud-director/vcloud-basic-auth.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
   name: vcloud-basic-auth
   namespace: kube-system
 data:
-  username: {{ .Credentials.VCD_USER | b64enc }}
-  password: {{ .Credentials.VCD_PASSWORD | b64enc }}
+  username: {{ default "" .Credentials.VCD_USER | b64enc }}
+  password: {{ default "" .Credentials.VCD_PASSWORD | b64enc }}
   refreshToken: {{ default "" .Credentials.VCD_API_TOKEN | b64enc }}
```

Comment:
I asked @ahmedwaleedmalik about this change, and we think we should drop fields that are not used instead of doing this. For example, if `username` and `password` are not used, we should leave only `refreshToken`, same other way around. Can you please update the PR to reflect this requirement?

Reply:
Most recent change should do the trick